developer/src/kmc-keyboard/src/util/util.ts

```diff
@@ -54,3 +54,25 @@ export function allUsedKeyIdsInLayers(layersList : LKLayers[] | null): Set<strin
   }
   return s;
 }
+
+/**
+ * Determine modifier from layer info
+ * @param layer layer obj
+ * @returns modifier
+ */
+export function translateLayerAttrToModifier(layer: LDMLKeyboard.LKLayer) : number {
+  const {id, modifier} = layer;
+  if(id == 'base') {
+    return constants.keys_mod_none;
+  }
```

Comment:
Is `base` a reserved id? Is this documented? Or should we be using only `modifier="none"` (or omitted `modifier` attribute)?

Reply:
i think it is reserved, but we should probably be using modifier only.

Reply:
will drop this code